lib/index.js

```diff
@@ -57,7 +57,12 @@ function render({ filename, files, metadata, engineOptions }) {
 
       // Transform the contents
       debug(`rendering ${ext} extension for ${filename}`);
-      file.contents = transform.render(file.contents, engineOptions, locals).body;
+      try {
+        file.contents = transform.render(file.contents, engineOptions, locals).body;
+      } catch (err) {
+        err.message = `${filename}: ${err.message}`;
+        throw err;
```

Comment:
I'd prefer using the reject callback here instead of throwing (see the Promise constructor)